### DISCUSSION TITLE
KRW currency dose not support decimal number

### DISCUSSION THREAD

**Discussion by ezur1** [dm4t2/vue-currency-input#356] [Q&A] [answered]
:options="{
        currency: KRW,
        locale: en-US,
      }"
when I use these input options i can not add "." to insert a decimal number.
**please help**

**Comment by dm4t2** [comment, marked as answer]
This is intended because KRW has no decimal digits:
<img alt="image" src="https://user-images.githubusercontent.com/11592259/202035153-ec742d3e-b342-4289-86a8-cd9759bc7941.png">

**Reply by ezur1** [reply]
thanks! maybe vue-currency-input has an option to not use currency but only use format number?
